pkgs/applications/editors/neovim/default.nix

```diff
@@ -108,6 +108,7 @@ let
                 $out/bin/nvim
     '' + optionalString withPython ''
       ln -s ${pythonEnv}/bin/python $out/bin/nvim-python
+      ln -s ${pythonEnv}/bin/python $out/bin/python2
```

Comment:
@steveeJ if you get vimpreviewpandoc to work and are stuck only with this ping me on irc, i can give you a helping hand fixing this PR.